docs/tutorial/working-with-frames-and-series/index.md

```diff
@@ -113,13 +113,17 @@ The `medications.date` column is an event series; it has many rows per patient.
 9 | 9 | 2015-03-14
 ```
 
-## Transform input data into output data
+## Transform tables into a dataset
 
-ehrQL tables, such as `patients` and `medications`, contain input data.
+Tables, such as `patients` and `medications`, contain either real or dummy level two data.
```

Comment:
Is it a distraction to introduce the security levels here?

Reply:
If we do talk about them, we should call them eg "Level 2" for consistency elsewhere.

Reply:
Changed.

Reply:
This is clearer, thanks.